static/openapi/vwrs-spec.json

```diff
@@ -467,6 +467,70 @@
             "example": "test-service-us-east.example.io"
           }
         }
+      },
+      "GetStatusResponse": {
+        "type": "object",
+        "properties": {
+          "avg_waiting_time": {
+            "type": "number",
+            "format": "float",
+            "description": "The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).",
```

Comment:
@joubert-macrometa, what is the standard for indicating time units? In this edit, we do it two different ways:

- Blah blah blah **(in time unit)**.
- Blah **in time unit** blah blah blah.